text/0000-project-groups.md

```diff
@@ -0,0 +1,328 @@
+Group terminology for organising and working on areas they are interested in,
+independent of The Rust Programming Language's Working Groups. It's great that
+we have a community able to self organise in this way, however it has led to
+some confusion over who is supporting these efforts, and whether they're
+considered _"official"_ Working Groups.
+
+This RFC aims to provide clarity by providing new distinct terminology that
+matches the intent behind "Team Working Groups", as well as codify some of the
+processes that these groups have been using to help facilitate creating
+new groups.
+
+[rust-lang#54445]: https://github.com/rust-lang/rust/issues/54445
+
+# Guide-level explanation
+
+To address this confusion this RFC proposes switching from using "Team Working
+Group" in favour of "Project Group". This would serve as a catch all term for
+the sub teams of the existing teams to organise around specific efforts, such
+as certain project or effort in the Rust team. Project Groups are led
+by "Shepherds".
+
+## Life-cycle of a Project Group
+
+This is a high level overview of the complete process of a project group. While
+the flow is built around project groups, we expect that working groups would
+follow a similar process with only minor specifics changed. E.g. A working
+group does not have to find a liaison.
+
+<p align="center">
+    <img src="../resources/project-group-workflow.png"
+        alt="A flow diagram showing each step of a project group"
+        height="800px">
+    <p align="center">Figure 1. Project Group Lifecycle</p>
+</p>
+
+### Steps
+
+1. Exploratory period.
+
+- Initial discussions of the problem area.
+- Write a short motivation for the project.
+- Find a person from the relevant team who's willing to act as a liaison.
+  - Typically you can find someone by creating a post on [internals] or
+    pinging specific people from team to gauge their interest.
+  - You may not always be able to find someone who is willing to act as
+    liasion. It's up to each team to decide how many new efforts
+    they'll have the bandwidth for, which may at times be none.
+
+2. Obtain the consensus of the team to create group.
+
+- Specify the liaison, and shepherd(s).
+- Write a short motivation, and some notes on possible solutions.
+- How consensus is reached would vary from team to team, some would require an
+  RFC while others could decide in a meeting. (See [Future Work](#future-work))
+
+3. Create infrastructure for group.
+
+- GitHub repository under `rust-lang` for hosting work and discussions, such
+  as for draft RFCs.
+- A Discord channel or a Zulip stream for communication.
+- Project group in [`rust-lang/team`], as well as a team on GitHub, for
+  handling permissions.
+
+4. Create a post on the Inside Rust blog announcing creation of the group. Be
+   sure to include the following information.
+  - An introduction
+  - The charter (either linked or inlined)
```

Comment:
We've not mentioned a charter up until this point I think, or at least not in the textual form -- I don't see it in the image but I can't grep that. Is this intended to be the same as the "short motivation"? If more elaborate, we should specify what a charter is and what it should include.